src/model.cpp

```diff
@@ -15,15 +15,36 @@
 //*****************************************************************************
 #include "model.hpp"
 
+#include <filesystem>
 #include <map>
 #include <memory>
 #include <sstream>
 #include <utility>
 
 #include "customloaders.hpp"
+#include "localfilesystem.hpp"
+#include "modelmanager.hpp"
```

Comment:
I would like to avoid such includes if possible. This means that with every change in modelmanager.hpp file you need to recompile model.cpp as well incrementally which adds to the lead time during development.

Reply:
Possibly filesystem_utils.hpp?